assets/dev/scss/helpers/variables.scss

```diff
@@ -138,6 +138,7 @@ $editor-background: $gray-bg;
 $editor-background-light: #f1f3f5;
 $editor-background-lighter: #fcfcfc;
 
+$editor-dark-blue: #010051;
```

Comment:
Is it supposed to be a general color or a specific one for the expert badge? If it's specific, change its name accordingly

Reply:
Fixed found this one on the var list: `$brand-color-royal-blue`